erro = 0
    
def set_erro(e):
    global erro
    print(e)
    erro = e